proto/umee/ibctransfer/v1/ibc_rate_limit.proto

```diff
@@ -0,0 +1,52 @@
+syntax = "proto3";
+package umee.ibctransfer.v1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+import "cosmos_proto/cosmos.proto";
+
+option go_package = "github.com/umee-network/umee/v3/x/ibctransfer";
+
+// RateLimit is format of ibc rate limit
+message RateLimit {
+  // ibc_denom defines the ibc denom
+  string ibc_denom = 1;
+  // expired_time defines rate limit expires time based on quota_duration
+  google.protobuf.Timestamp expired_time = 2 [
+    (gogoproto.stdtime)  = true,
+    (gogoproto.nullable) = true,
+    (gogoproto.moretags) = "yaml:\"expired_time\""
+  ];
+  // outflow_sum defines the sum of outflow tokens from ibc-transfer
+  string outflow_sum = 3 [
+    (cosmos_proto.scalar)  = "cosmos.Dec",
```

Comment:
Why using Dec rather than Int?

Reply:
because price is in decimals

Reply:
I know, but maybe it's enough to trace USD as Ints (and round up)? Otherwise let's update the comment and say that this is a USD price sum.